components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/StorageFacetImpl.java

```diff
@@ -95,7 +108,10 @@ public StorageFacetImpl(final BlobStoreManager blobStoreManager,
                           final @Named(ComponentDatabase.NAME) Provider<DatabaseInstance> databaseInstanceProvider,
                           final BucketEntityAdapter bucketEntityAdapter,
                           final ComponentEntityAdapter componentEntityAdapter,
-                          final AssetEntityAdapter assetEntityAdapter)
+                          final AssetEntityAdapter assetEntityAdapter,
+                          final ClientInfoProvider clientInfoProvider,
+                          final Map<String, ContentValidator> contentValidators,
+                          final DefaultContentValidator defaultContentValidator)
```

Comment:
The default validator is available via `contentValidators.get(DefaultContentValidator.NAME)` so this parameter could be removed.